content/en/serverless/aws_lambda/configuration.md

```diff
@@ -86,6 +89,39 @@ First, [install][1] Datadog Serverless Monitoring to begin collecting metrics, t
 - [Troubleshoot](#troubleshoot)
 - [Further Reading](#further-reading)
 
+## Enable Threat Detection to observe attack attempts
+
+Get alerted on attackers targeting your serverless applications and respond quickly. 
+
+To get started, first ensure that you have [tracing enabled][40] for your functions.
+
+To enable threat monitoring, simply add the following environment variables depending on the language:
+   ```yaml
+   environment:
+     DD_SERVERLESS_APPSEC_ENABLED: true
+   ```
+   For **Go functions only** also add:
+   ```yaml
+   environment:
+     DD_UNIVERSAL_INSTRUMENTATION: true
+   ```
+   For **NodeJS or Python functions** also add:
+   ```yaml
+   environment:
+     DD_EXPERIMENTAL_ENABLE_PROXY: true
+     AWS_LAMBDA_EXEC_WRAPPER: /opt/datadog_wrapper
+   ```
+
+Redeploy the function and invoke it. After a few minutes, it appears in [ASM views][3].
+
+[3]: https://app.datadoghq.com/security/appsec?column=time&order=desc
+
+To see Application Security Management threat detection in action, send known attack patterns to your application. For example, send an HTTP header with value `acunetix-product` to trigger a [security scanner attack][5] attempt:
```

Comment:
This link is currently opening a blog post, which seems to match the other `[5]` reference link on line `223`. Is there a new link to go with `security scanner attack`?